src/app.ts

```diff
@@ -32,6 +32,14 @@ prClient.getPullRequests().then(prs => {
         const tableCellRepo = document.createElement("td");
         tableCellRepo.innerText = pr.repo;
         tableRow.appendChild(tableCellRepo);
+        // Base cell
+        const tableCellBaseBranch = document.createElement("td");
+        tableCellBaseBranch.innerHTML = "<a href='" + (pr.baseUri + pr.projectName + "/_git/" + pr.repo + "?version=GB" + pr.baseBranch) + "' target='_top'>#" + pr.baseBranch + "</a>";
```

Comment:
A branch name can contain characters such as `/` within it. Probably need to do some URI escaping on there too. I can confirm that in one of our repositories the URL looks like `https://corp.visualstudio.com/ProjectName/_git/Cool%20Business%20App?version=GBbug%2Ffix-the-things`

Reply:
This is the worst URL I could make. I can't even fetch this branch!

```
/Test%20_Project(%5EName.!)/_git/Test%20_Repo(%5EName.!)?version=GB'%2F%3Einjection
```

Reply:
Assuming the values in `.targetBranch` and `.baseBranch` are the unescaped names, some quick testing with encodeURIComponent shoes that it should do the trick here:

```js
encodeURIComponent("'/>injection") // "'%2F%3Einjection"
encodeURIComponent("Test _Project(^Name.!)") // "Test%20_Project(%5EName.!)"
encodeURIComponent("pb&j") // "pb%26j"
```

Reply:
Good call, I will sprinkle encodeURIComponent where needed.